### DISCUSSION TITLE
How we can connect Multi AutoGPT Agents together

### DISCUSSION THREAD

**Discussion by abdalhadi21** [FlowiseAI/Flowise#1693] [Q&A]
Hi!
How we can connect Multi AutoGPT Agents together! Assuming we have 3 or more Agents and each has different responsibilities.
![Screenshot 2024-02-07 at 16 44 44](https://github.com/FlowiseAI/Flowise/assets/119605852/63537096-c149-4440-b634-f73492954673)

**Comment by sdadesky** [comment]
I have not tried this (very focused on crewai recently). But I did notice they have if / else functionality. Adding that to variables would allow you to user variable flags:  if process1 is complete and process3 is complete run process 4...
